packages/pdfkit/src/font/data/compressData.js

```diff
@@ -1,6 +1,11 @@
 import fs from 'fs';
+import url from 'url';
 import { basename, extname } from 'path';
-import { parse } from '../afm';
+// This file is ran directly with Node - needs to have .js extension
```

Comment:
Don't we always need .js extension with ESM?

Reply:
Yes and no! In most cases, react-pdf is pre-bundled. When bundled, module resolution is already done by the bundler and these rules no longer apply. 

I WOULD do this if I were you sooner or later, see "Adding file extensions in select places" in original post for rationale.